messages/views.py

```diff
@@ -159,23 +163,18 @@ def new_message(request, username=None, message_id=None):
                 
                 body = message.body.body.replace("\r\n", "\n").replace("\r", "\n")
                 body = ''.join(BeautifulSoup(body).findAll(text=True))
-                body = "\n".join([(">" if line.startswith(">") else "> ") + "\n> ".join(wrap(line.strip(),60)) for line in body.split("\n")])
+                body = "\n".join([(">" if line.startswith(">") else "> ") + "\n> ".join(wrap(line.strip(), 60))
+                                  for line in body.split("\n")])
                 body = "> --- " + message.user_from.username + " wrote:\n>\n" + body
 
                 subject = "re: " + message.subject
                 to = message.user_from.username
 
-                if request.user.profile.num_sounds:
-                    form = MessageReplyFormNoCaptcha(initial=dict(to=to, subject=subject, body=body))
-                else:
-                    form = MessageReplyForm(initial=dict(to=to, subject=subject, body=body))
+                form = form_class(initial=dict(to=to, subject=subject, body=body))
```

Comment:
I typically use dict literals here instead of the class syntax:
`{"to": to, "subject": subject}`

Reply:
Addressed in 629ff4174be55a9d770a0dadc3ad36be502233fc